docs/clients/coil_temperature_estimator.rst

```diff
@@ -83,9 +83,9 @@ A minimal working example for the CoilTemperatureEstimatorClient is:
     % Make a communication interface object
     com = MessageInterface(’COM18’,115200);
     % Make a CoilTemperatureEstimatorClient object with obj_id 0
-    tes = CoilTemperatureEstimatorClient(’com’,com);
+    coilTemperatureEstimator = CoilTemperatureEstimatorClient(’com’,com);
```

Comment:
I agree tes is a pretty bad variable name.  
com is an object variable.
coilTemperatureEstimator is an object variable.
coil_temp is a normal variable.

The Matlab style guide says "Write variable names in mixed case starting with lower case."
The Matlab style guide says "Structure names should begin with a capital letter."
It does not say what to do with class objects.  Are they variables (they have a type of Class x)?  Are they structures (they can be . indexed)?

They have different styles.